test/models/github-test.js

```diff
@@ -0,0 +1,15 @@
+const assert = require('assert')
+const fs = require('fs')
+const path = require('path')
+
+const GitHub = require('../../src/models/github')
+
+describe('GitHub', () => {
+  describe('getToken', () => {
+    it('returns contents of .env file', () => {
+      const tokenPath = path.join(__dirname, '..', '..', '.env')
```

Comment:
What do you think about faking this, mocking it somehow? Forcing the test envs to have an `.env` file seems uncecessary.